pkg/agent/controller/networkpolicy/packetin.go

```diff
@@ -224,3 +233,45 @@ func getPacketInTableID(pktIn *ofctrl.PacketIn) uint8 {
 	}
 	return tableID
 }
+
+func getCTMarkValue(matchers *ofctrl.Matchers) uint32 {
+	ctMark := matchers.GetMatchByName("NXM_NX_CT_MARK")
+	if ctMark == nil {
+		return 0
+	}
+	ctMarkValue, ok := ctMark.GetValue().(uint32)
+	if !ok {
+		return 0
+	}
+	return ctMarkValue
+}
+
+func getCTNwDstValue(matchers *ofctrl.Matchers) *net.IP {
```

Comment:
I don't think there is a strong reason to return a pointer here

Reply:
Now that I merged #5532, you may have some conflicts to resolve. And this function should return a `netip.Addr` value.

Reply:
Updated it. Thanks.